velox/exec/Window.cpp

```diff
@@ -171,8 +194,10 @@ void Window::callResetPartition() {
   currentPartition_ = nullptr;
   if (windowBuild_->hasNextPartition()) {
     currentPartition_ = windowBuild_->nextPartition();
-    for (int i = 0; i < windowFunctions_.size(); i++) {
-      windowFunctions_[i]->resetPartition(currentPartition_.get());
+    if (currentPartition_) {
```

Comment:
hasNextPartition should always return true if there is a new partition. currentPartition_ should not be nullptr.

Reply:
Removed.